models/tfidf_model/tfidf.py

```diff
@@ -0,0 +1,102 @@
+import math
+from collections import Counter
+import re
+import numpy as np
+from tfidf_model.config import *
+from tfidf_model.utils import *
+print(PATH)
```

Comment:
this could go to 

```
if __name__ == "__main__":
   ...
```

block, but probably not a good idea to have it in here.